### ISSUE TITLE
Import throws CallbackOnCollectedDelegate

### ISSUE THREAD

**Issue by Tomiinek** [migueldeicaza/TensorFlowSharp#135]
Hello, I can not resolve why I am getting an `CallbackOnCollectedDelegate exception` from the `Import`  method while I am calling the `DoSomeMagic` method twice (even while importing the same model or two differenct models).  This code causes the problem:

``` c#
public void DoSomeMagic(string path) {
   var graph = new TFGraph();
   var model = File.ReadAllBytes(path);
   graph.Import(model, "");
   // this does not make a change -> graph.Dispose();
}

**Comment by doxakis** [comment]
Hi,

I can reproduce it with the 1.2.2 version.
However, it is fixed with the 1.3.0-pre1.

Thanks!

**Comment by migueldeicaza** [comment]
Closing per the above.

For now, upgrade to the -pre build, and once I get a chance, I will upgrade it.
